modules/sugar/src/test/ts/browser/CssTest.ts

```diff
@@ -39,7 +39,7 @@ UnitTest.test('CssTest', () => {
     Css.copy(c, c2);
     Css.copy(m, c2);
 
-    // NOTE: Safari and Firefox 71+ seems to support styles for math ml tags, so the Css.copy(m, c2) clobbers the previous style
+    // NOTE: Safari, Firefox 71+ and Chrome 102 seem to support styles for math ml tags, so the Css.copy(m, c2) clobbers the previous style
```

Comment:
```suggestion
    // NOTE: Safari, Firefox 71+ and Chrome 102+ seem to support styles for math ml tags, so the Css.copy(m, c2) clobbers the previous style
```
This needs to be future proof as well.

Reply:
@tiny-ben-tran looks like only half of the suggestion was made here? Admittedly I did edit that immediately after submitting the comment so you may have missed it because of that.